components/config/prefs.rs

```diff
@@ -577,7 +577,14 @@ mod gen {
                 },
                 mime: {
                     sniff: bool,
-                }
+                },
+                tls: {
+                    /// Ignore `std::io::Error` with `ErrorKind::UnexpectedEof` received if the connection
+                    /// is closed without a close_notify.
+                    ///
+                    /// Used for tests because WPT server doesn't propely close the TLS connection.
+                    ignore_unexpected_eof: bool,
```

Comment:
My preference would be to fix the WPT server instead of adding more test-only behaviour differences, but I think it makes sense to handle that in a separate followup issue.

Reply:
Yeah, fixing WPT server is preferable. I added a TODO note to remove it when that happens.